collect_data.py

```diff
@@ -12,17 +12,22 @@
 # Basic headers for GitHub public API
 HEADERS = {"Accept": "application/vnd.github+json", "User-Agent": "PR-Watcher"}
 
-# Search queries - tracking merged PRs
+# Search queries - tracking merged PRs and non-draft PRs
 Q = {
     "is:pr+head:copilot/": "copilot_total",
+    "is:pr+head:copilot/+-is:draft": "copilot_ready",
```

Comment:
Re-run `collect_data.py` and generate chart script to ensure these updates are reflected in the generated code in the repo.